keras_nlp/tokenizers/word_piece_tokenizer.py

```diff
@@ -112,11 +112,6 @@ class WordPieceTokenizer(tokenizer.Tokenizer):
         split: If true, input will be split according to `split_pattern`
             and `keep_pattern`. If false, input should be split before calling
             the layer.
```

Comment:
Maybe we should update the split description to something like "split: If true, input will be split by whitespace and punctuation, and punctuations would be kept. If false, input should be split before calling the layer."?